spring-cloud-contract-verifier/src/test/groovy/org/springframework/cloud/contract/verifier/builder/MockMvcMethodBodyBuilderSpec.groovy

```diff
@@ -559,7 +559,7 @@ class MockMvcMethodBodyBuilderSpec extends Specification implements WireMockStub
 		builder.appendTo(blockBuilder)
 		def test = blockBuilder.toString()
 		then:
-		test.contains('get("/users?limit=10&offset=20&filter=email&sort=name&search=55&age=99&name=Denis.Stepanov&email=bob@email.com")')
+		test.contains('get("/users?limit={limit}&offset={offset}&filter={filter}&sort={sort}&search={search}&age={age}&name={name}&email={email}","10","20","email","name","55","99","Denis.Stepanov","bob@email.com")')
```

Comment:
see c7b1d274b5ecb179ff0dab63f73064f84631bf8a changed it to use the correct method for queryParameters

see also test or code example

```

ResponseOptions response = given().spec(request)
					.queryParam("limit","10")
					.queryParam("offset","20")
					.queryParam("filter","email")
					.queryParam("sort","name")
					.queryParam("search","55")
					.queryParam("age","99")
					.queryParam("name","Denis.Stepanov")
					.queryParam("email","bob@email.com")
					.get("/users");

```